main/src/main/java/org/mobilitydata/gtfsvalidator/table/JsonFileLoader.java

```diff
@@ -0,0 +1,86 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.mobilitydata.gtfsvalidator.notice.IOError;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.validator.ValidatorProvider;
+
+public class JsonFileLoader extends TableLoader {
```

Comment:
This looks no generic enough to name it as `JsonFileLoader`. I suggest to rename it to GeoJsonFileLoader or make the implementation generic enough to cover any JSON descriptor.